### DISCUSSION TITLE
why no voice "zh-CN-XiaoxiaoMultilingualNeural"

### DISCUSSION THREAD

**Discussion by liqingjun123** [rany2/edge-tts#224] [Q&A] [answered]
"zh-CN-XiaoxiaoMultilingualNeural" was added in Azure AI.

**Comment by rany2** [comment, marked as answer]
The voices available in Azure's TTS offering and Microsoft Edge is different but at least so far, Microsoft seems to allow users to use all the voices from Azure **_eventually_** with some delay. At least for the moment, you could use another multilingual voice:

```
$ edge-tts --list-voices|grep Multilingual
Name: de-DE-FlorianMultilingualNeural
Name: de-DE-SeraphinaMultilingualNeural
Name: en-US-AndrewMultilingualNeural
Name: en-US-AvaMultilingualNeural
Name: en-US-BrianMultilingualNeural
Name: en-US-EmmaMultilingualNeural
Name: fr-FR-RemyMultilingualNeural
Name: fr-FR-VivienneMultilingualNeural
```

**Reply by rafferty90** [reply]
@rany2 how may I use Azure nb-NO-Iselin voice on edge-tts?
